mediapipe/models/ovms/config_holistic.json

```diff
@@ -35,6 +35,12 @@
                 "base_path":"./pose_landmark_full",
                 "plugin_config": {"PERFORMANCE_HINT": "LATENCY", "AFFINITY": "NUMA"}
             }
+        },
+        { "config": {
```

Comment:
Maybe this shouldn't be now config_holistic.json?

Either we should prepare one big config containing all required models at the cost of app startup time or we should have specific jsons per example.

Reply:
Will change in the next task.